tests/baselines/reference/thisShadowingErrorSpans.errors.txt

```diff
@@ -0,0 +1,16 @@
+tests/cases/compiler/thisShadowingErrorSpans.ts(5,13): error TS2683: 'this' implicitly has type 'any' because it does not have a type annotation.
+
+
+==== tests/cases/compiler/thisShadowingErrorSpans.ts (1 errors) ====
+    class C {
+        m() {
+            this.m();
+            function f() {
+                this.m();
+                ~~~~
+!!! error TS2683: 'this' implicitly has type 'any' because it does not have a type annotation.
+!!! related TS2735 tests/cases/compiler/thisShadowingErrorSpans.ts:4:18: There is a 'this' type shadowed by this container.

```

Comment:
As a side note...I think we need better baselines for related information.. Just looking at this baseline first thing I see from message is I don't think message clearly states that its talking about `f` but then later I saw span. May be it would be good idea  to use pretty formatting of related information instead because that's how its going to emerge anyways?

Reply:
If you set the pretty flag in the test options, it actually does get recorded like that. i was told that it wasn't typically necessary though, esp. since pretty also has a bunch of formatting characters which don't translate to plaintext well.